sdk/ai/azure-ai-inference/samples/sample_chat_completions_with_tracing.py

```diff
@@ -0,0 +1,168 @@
+# ---------------------------------------------------------
```

Comment:
Please update the README.md in the samples folder to mention this sample

Reply:
fixed